### DISCUSSION TITLE
Know HA version where Gree Custom Integration work without any problem

### DISCUSSION THREAD

**Discussion by rapi3** [rapi3/HA-OS-Gree#3] [General]
Latest HA version confirmed and reported integration work:
| Name | Version |
| -------- | ------- |
| Core | 2025.4.2 |
| Supervisor | 2025.04.0 |
| Operating System | 15.1 |
| Frontend | 20250411.0 |

edit: Updated in 20-04-2025

**Comment by radaron** [comment]
Hey @rapi3,
Thank you!
I am using this integration, what works excellent with the following HA versions:

| Name | Version |
| -------- | ------- |
| Core | 2025.3.1 |
| Supervisor | 2025.03.2 |
| Operating System | 14.2 |
| Frontend | 20250306.0 |

**Reply by rapi3** [reply]
Thank you for feedback.
